src/reanimated2/shareables.ts

```diff
@@ -149,6 +149,14 @@ export function makeShareableCloneRecursive<T>(
               value.__stackDetails
             );
             delete value.__stackDetails;
+          } else if (value.__stackDetails) {
+            // Detected debug version of the worklet in release bundle. This
+            // might lead to unexpected issues or errors. Probably one of user
+            // dependencies provided transpiled code with debug version of the
+            // Reanimated plugin.
+            throw new Error(
+              '[Reanimated] Using dev bundle in a release app build is not supported.'
```

Comment:
We might want to give more information on how to troubleshoot this problem, i.e. clear cache or something.

Reply:
Sure thing 👍